src/test/codegen/packed.rs

```diff
@@ -54,9 +54,6 @@ pub struct PackedPair(u8, u32);
 // CHECK-LABEL: @pkd_pair
 #[no_mangle]
 pub fn pkd_pair(pair1: &mut PackedPair, pair2: &mut PackedPair) {
-    // CHECK: [[V1:%[a-z0-9]+]] = load i8, i8* %{{.*}}, align 1
```

Comment:
Is the case this test was intended to check unreachable?

Reply:
Now, yeah, `ScalarPair` only works by having the placement of the second scalar at an ABI-aligned offset, so it's incompatible with packing.